.bazelrc

```diff
@@ -15,7 +15,15 @@ build --test_output=errors # Prints log file output to the console on failure
 
 # By default do not build the tests for sources-with-prebuilt-binaries,
 # because it takes quite some time. They will only run on CI
-build --deleted_packages tests/ios/frameworks/sources-with-prebuilt-binaries
+#
+# By default do not run the tests under the static_framework_generator package, 
+# as it requires that tools/static_framework_generator:TargetXIBStaticFrameworkGenerator
```

Comment:
why can't it be built as a dependency?

Reply:
The rule `tools/static_framework_generator:TargetXIBStaticFrameworkGenerator` generates a framework at the location `tests/ios/static_framework_generator/bazel-generated`.

I decided to test the rule by consuming the framework and using it in a test target (`tests/ios/static_framework_generator:TargetXIBTestsLib`), . To consume the framework I am using `apple_static_framework_import` (`tests/ios/static_framework_generator:TargetXIBStaticFramework`).

The rule `apple_static_framework_import` requires that the framework is present on disk at analysis time. That is why it cant be built as a dependency, and instead it has to be done in two steps:

1. Generate the framework by running `tools/static_framework_generator:TargetXIBStaticFrameworkGenerator`
2. Test that the generated framework can be consumed without issues

I am not aware of a rule similar to `apple_static_framework_import` that does not require the framework to be ready at analysis time.